### DISCUSSION TITLE
Covid infection survey variables

### DISCUSSION THREAD

**Discussion by rzepnk** [opensafely/documentation#1191] [General]
Hello. I need to add 2 new variables to our study population from the Covid Infection Survey, those are imd_stamp and rural_urban. I had a look at the [cohort-extractor file ](https://github.com/opensafely-core/cohort-extractor/blob/main/cohortextractor/ons_cis_utils.py) and I can't seem to find them there. Does that mean the variables are not available for us to use? 

Thanks

**Comment by inglesp** [comment]
Hi @rzepnk -- I've followed up on Slack.
